### ISSUE TITLE
Proxy support

### ISSUE THREAD

**Issue by BlackFlag-Security** [RUB-NDS/TLS-Attacker-BurpExtension#4]
The add-on currently doesn't use the proxy/socks settings configured for Burp

**Comment by jurajsomorovsky** [comment]
Yes, it uses the default socket implementation. Using BurpSuite socket would need many code changes.
What would be the motivation to use it? (considering the plugin is used to evaluate TLS)

**Comment by BlackFlag-Security** [comment]
I used the plugin to test a staging site yesterday, it is non-public and my jump box had been IP whitelisted, but not my local IP, I was Burp socks proxy to access the site via the jump box. The plugin didn't work because of this. 
I ended up getting it working by using SSH -L to port-forward and then ran the plugin against localhost.
I agree this is kind of an edge case, and if it'd a lot of work to change, I understand not prioritizing it.

**Comment by jurajsomorovsky** [comment]
Ok, I am glad that you found a different way to solve this issue. 

We will discuss this issue in the next meeting
